lib/api/v3/work_packages/work_package_representer.rb

```diff
@@ -324,7 +324,14 @@ def initialize(model, current_user:, embed_links: false)
         property :description,
                  exec_context: :decorator,
                  getter: ->(*) {
-                   ::API::Decorators::Formattable.new(represented.description, object: represented)
+                   type = represented&.type
+                   description = represented&.description
+
+                   if !description.present? && type&.description
+                     ::API::Decorators::Formattable.new(type.description, object: represented)
+                   else
+                     ::API::Decorators::Formattable.new(description, object: represented)
+                   end
```

Comment:
If I am not mistaken, this will lead to all work packages not having a description receiving the `type.description` (if that is defined) even if they are already persisted which is not what is intended, I believe.

Additionally, I think that this kind of business logic does not belong into the representer, but rather into the service layer.

Reply:
https://github.com/opf/openproject/blob/dev/app/services/work_packages/set_attributes_service.rb#L85 is a good place